lib/ougai/logging.rb

```diff
@@ -84,6 +83,22 @@ def trace?
       level <= TRACE
     end
 
+    # Log any one or more of a message, an exception and structured data as specified log level.
+    # If the block is given for delay evaluation, it returns them as an array or the one of them as a value.
+    # @param severity [Integer] The log level.
+    # @param message [String] The message to log. Use default_message if not specified.
+    # @param ex [Exception] The exception or the error
+    # @param data [Object] Any structured data
+    # @yieldreturn [String|Exception|Object|Array] Any one or more of former parameters
+    # @return [Boolean] true
+    def add(severity, *args)
+      severity ||= UNKNOWN
```

Comment:
The same complementation as https://github.com/ruby/logger/blob/master/lib/logger.rb#L460